src/interfaces.ts

```diff
@@ -9,12 +9,14 @@ import type {
   TorusVerifierResponse,
   UX_MODE_TYPE,
 } from "@toruslabs/customauth";
+// TODO: move the types to a base class for both dkls and frost in future
+import type { tssLib } from "@toruslabs/tss-dkls-lib";
 import BN from "bn.js";
 
 import { FactorKeyTypeShareDescription, TssShareType, USER_PATH, WEB3AUTH_NETWORK } from "./constants";
 
 export type CoreKitMode = UX_MODE_TYPE | "nodejs" | "react-native";
-
+export type TssLibType = typeof tssLib;
```

Comment:
maybe this can be `(typeof tssLibFrost | typeof tssLibDKLS)`?

Reply:
yes fixed